tests/lwip/main.c

```diff
@@ -79,7 +66,7 @@ static const shell_command_t shell_commands[] = {
 #ifdef MODULE_SOCK_UDP
     { "udp", "Send UDP messages and listen for messages on UDP port", udp_cmd },
 #endif
-    { "ifconfig", "Shows assigned IP addresses", ifconfig },
+    { "v6addr", "Shows IPv6 addresses (for tests)", v6addr },
```

Comment:
Can't we just rewrite the test to use `ifconfig` instead? I certainly could help with that.

Reply:
Sure, the only slight complication is from the multiline printing. But since the current code doesn't worry about there being more than one interface it should be ok. I will give it a try.

Reply:
This causes a test failure for the tcp test since the client address is `fe80:0:0:0:e463:cff:fee1:47a3` but the address reported by the tcp code is `fe80::e463:cff:fee1:47a3` (note change in zero compression). Maybe it is better to use the same ip-to-string functions in the ifconfig printing after all.

Reply:
Yeah... the `fe80:0:0:0:e463:cff:fee1:47a3` is, while still correct, very non-standard. An alternative approach could be to go via `python`'s ipaddress module for parsing the addresses and generating the appropriate output :sweat_smile:

Reply:
For reference, this is the module I was talking about: https://docs.python.org/3/library/ipaddress.html

Reply:
Thanks, I used that to convert the address to the minimal form.